### PR TITLE
(update) ng 17

### PR DESCRIPTION
**Pull request by StefanAleksik**

### CODE REVIEW

**Comment by andreaslarssen** [comment]
Thanks! Will release new version some time next week I hope.

File: package.json [hunk 1]
```diff
@@ -11,45 +11,45 @@
...
-    "@angular-devkit/build-angular": "~16.0.1",
-    "@angular-eslint/builder": "16.0.1",
-    "@angular-eslint/eslint-plugin": "16.0.1",
-    "@angular-eslint/eslint-plugin-template": "16.0.1",
-    "@angular-eslint/schematics": "16.0.1",
-    "@angular-eslint/template-parser": "16.0.1",
-    "@angular/cli": "~16.0.1",
-    "@angular/compiler-cli": "~16.0.1",
-    "@angular/language-service": "~16.0.1",
+    "@angular-devkit/build-angular": "~17.3.2",
```
Comment (andreaslarssen) [inline package.json:27]: Could you change to ~17.0.0 on all Angular packages?

Comment (StefanAleksik) [reply package.json:27]: Will do.

**Comment by andreaslarssen** [comment]
Sorry about the delay, been out with a cold. Will try and fix tomorrow

**Comment by andreaslarssen** [comment]
@StefanAleksik  New version with A17 support is out (4.0.0). Thanks for the PR!

**Pull request merged**